### PR TITLE
Ziafazal/wl-1771: Added fullscreen support fo document and video viewer

### PR DESCRIPTION
**Pull request by ziafazal**
This PR has changes to provide fullscreen support fo document and video viewer.

PS: Fullscreen API is not support on [iOS safari](https://caniuse.com/#feat=fullscreen)

@bill-filler for some reason my last commit on PR #69 was not shown. Created this PR to have overcome it.

### CODE REVIEW

**Comment by bill-filler** [comment]
should hide the fullscreen buttons in mobile because they do not work

**Comment by bill-filler** [comment]
@tuchfarber 
can you review the css changes please?

**Comment by bill-filler** [comment]
@ziafazal 
the height of videos and documents in mobile is now much shorter then before. Don't think it's good especially for documents because it doesn't fit a full page anymore

**Comment by ziafazal** [comment]
@bill-filler I removed extra padding for video and document viewers. Attached screenshot to give you an idea of extra padding.
screenshot_20181023-122456
screenshot_20181023-122548

**Comment by tuchfarber** [comment]
I can review, but this majority of this work edits @BessieSteinberg 's work. She'll have a better idea of what the changes might affect.

**Comment by tuchfarber** [comment]
@bill-filler Originally we used the custom `responsive-video` class to better handle differences between mobile and desktop instead of `embed-responsive-16by9`. The `embed-responsive-16by9` should make the height of the container 9/16 of the width of the phone, which is shorter than we'd want for a document.

If that change wasn't needed for this feature, reverting the class change and the class's scss removal should fix the sizing issue.

**Comment by ziafazal** [comment]
@bill-filler @tuchfarber I have removed the `embed-responsive-16by9` class and using aspect ratio media queries to adjust padding in fullscreen mode.

File: src/components/PDFViewer/PDFViewer.scss [hunk 1]
```diff
@@ -2,3 +2,20 @@
...
+
+@media screen and (min-device-aspect-ratio: 16/10) {
+  .responsive-pdf::before{
+    padding-top: 62%;
+  }
+}
+
+@media screen and (min-device-aspect-ratio: 16/9) {
+  .responsive-pdf::before{
+    padding-top: 56%;
```
Comment (bill-filler) [inline src/components/PDFViewer/PDFViewer.scss]: change this to 85% as it makes it taller on desktop screens and fill most of the screen which is what we want

**Pull request merged**